src/main/java/core/basesyntax/data/ReportService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.data;

```

Comment:
pay attention to the packages, this service, mapper, should be moved to some separate package